docker/generic/start_proxy.py

```diff
@@ -540,6 +562,13 @@ def enforce_conflict_args(args):
     if len(port_flags) > 1:
         return "Multiple port flags {} are not allowed, please just use --listener_port flag".format(",".join(port_flags))
 
+    if args.ssl_minimum_protocol and args.ssl_minimum_protocol not in {"TLSv1.0", "TLSv1.1", "TLSv1.2", "TLSv1.3"}:
```

Comment:
Suggest you use this instead: https://docs.python.org/3/library/argparse.html#choices

Reply:
You can then also remove the values from the help text

Reply:
Thanks, will do in following PR, since there are some other flags also need to change.